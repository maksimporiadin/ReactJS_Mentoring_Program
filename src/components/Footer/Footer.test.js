import 'jsdom-global/register';
import React from 'react';
import { Footer } from './index';
import { shallow } from 'enzyme';

describe('Footer', () => {
    let wrapper;

    it('is renders', () => {
        wrapper = shallow(<Footer  />);
        console.log(wrapper)
        expect(wrapper).toMatchSnapshot();
    });
});