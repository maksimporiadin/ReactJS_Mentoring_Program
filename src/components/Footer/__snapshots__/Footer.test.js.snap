// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is renders 1`] = `
<footer
  className="footer"
>
  <Logo />
</footer>
`;
